workers/email/src/templates/emails/events-limit-almost-reached/html.twig

```diff
@@ -14,23 +14,23 @@
     <td align="center" style="padding: 15px 0;">
       <font color="#dbe6ff" style="font-size: 15px; text-align: center; color: #dbe6ff; letter-spacing: 0.4px;">
         <span style="vertical-align: middle; display: inline-block;">
-          Your plan limit is almost reached
+          Your limit of events is almost reached
```

Comment:
If `eventsCount` is bigger than `eventsLimit` — there shouldn't be an `almost` word because it looks strange 

<img width="595" alt="image" src="https://user-images.githubusercontent.com/3684889/218461790-305d2fa8-fc0d-4858-9b6c-66f3531d5120.png">

Reply:
same thing in Email subject

<img width="435" alt="image" src="https://user-images.githubusercontent.com/3684889/218462141-653eed0e-e173-4c33-8811-d912277ba63a.png">